tools/accuracy_checker/accuracy_checker/adapters/text_detection.py

```diff
@@ -998,3 +998,72 @@ def restore_rectangle(origin, geometry):
             new_p_1 = np.zeros((0, 4, 2))
 
         return np.concatenate([new_p_0, new_p_1])
+
+
+class CTCGreedySearchDecoder(Adapter):
+    __provider__ = 'ctc_greedy_search_decoder'
+    prediction_types = (CharacterRecognitionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'blank_label': NumberField(
+                optional=True, value_type=int, min_value=0, description="Index of the CTC blank label."
```

Comment:
you can set zero as default value, the code below shows that you do not allow usage without blank label at all, so you can avoid additional checks with such replacement
```suggestion
                optional=True, value_type=int, min_value=0, default=0, description="Index of the CTC blank label."
```

Reply:
BTW, why is ctc_beam_search_decoder with beam_size=1 and blank_label=0  unsuitable for you?

Reply:
> BTW, why is ctc_beam_search_decoder with beam_size=1 and blank_label=0 unsuitable for you?

Hi eaidova,
For our model, the class number (the dimension D is 1161) is much bigger than the english text recognition (D = 37). If our model uses the existing beam search decoder, the running will be very slow and hang... So we just use numpy.argmax to select the max index in the D dimension. The results on our side show for our model and task, the approach is effective and efficient enough.

Reply:
ok, thanks for explanation